src/components/encounter/ohri-encounter-form.component.tsx

```diff
@@ -455,8 +469,9 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
     if (field.fieldDependants) {
       field.fieldDependants.forEach(dep => {
         const dependant = fields.find(f => f.id == dep);
+        // console.log({ dependant, field, fields });
```

Comment:
```suggestion
       
```

Reply:
Seems we have a conficts @kajambiya have a look at that, Thanks!

Reply:
Doing some final edits. Wiring edited commit in a bit